kale/embed/mpca.py

```diff
@@ -14,6 +14,7 @@
 import warnings
 
 import numpy as np
+from scipy import linalg
```

Comment:
So is `scipy.linalg` better than `np.linalg`? If yes, `scipy` needs to be added to `setup.py` `install..._req..."